gateway/ln-gateway/tests/fixtures.rs

```diff
@@ -24,6 +27,10 @@ pub async fn fixtures() -> (
     let ln_params = LightningGenParams::regtest(fixtures.bitcoin_server());
     fixtures = fixtures.with_module(LightningClientGen, LightningGen, ln_params);
 
+    let wallet_params = WalletGenParams::regtest(fixtures.bitcoin_server());
+    let wallet_client = WalletClientGen::new(fixtures.bitcoin_client());
+    fixtures = fixtures.with_module(wallet_client, WalletGen, wallet_params);
```

Comment:
It's probably a bit confusingly named, but you don't need the Wallet module here. The wallet module handles peg-ins and peg-outs for on-chain bitcoin. The test you're implementing is just validating if the `info` command returns the balance of the federation (which in the case of tests, is the dummy module. In a real Fedimint it will be the Mint module).